### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by MaximAlien**
### Description

PR updates dependencies to Maps `v10.6.0`, Navigation Native `v106.0.0` and Common `v22.0.0`.

/cc @azarovalex

### CODE REVIEW

File: Sources/MapboxCoreNavigation/NativeHandlersFactory.swift [hunk 1]
```diff
@@ -53,7 +53,8 @@ class NativeHandlersFactory {
     
     lazy var navigator: MapboxNavigationNative.Navigator = {
         onMainQueueSync { // Make sure that Navigator pick ups Main Thread RunLoop.
-            LogConfiguration.setLoggingLevelForCategory("*", level: LoggingLevel.info)
+            let loggingLevel = NSNumber(value: LoggingLevel.info.rawValue)
+            LogConfiguration.setLoggingLevelForCategory("*", upTo: loggingLevel)
```
Comment (MaximAlien) [inline Sources/MapboxCoreNavigation/NativeHandlersFactory.swift:57]: Is this API really working? I've changed to other logging levels and do not see any difference (e.g. still getting debug related logs from Navigation Native regardless of provided `loggingLevel` value).

File: Package.swift [hunk 2]
```diff
@@ -24,8 +24,8 @@ let package = Package(
     dependencies: [
         .package(name: "MapboxDirections", url: "https://github.com/mapbox/mapbox-directions-swift.git", .exact("2.6.0-beta.2")),
         .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", from: "1.0.0"),
-        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "105.0.0"),
-        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.6.0-rc.1")),
+        .package(name: "MapboxNavigationNative", url: "https://github.com/mapbox/mapbox-navigation-native-ios.git", from: "106.0.0"),
+        .package(name: "MapboxMaps", url: "https://github.com/mapbox/mapbox-maps-ios.git", .exact("10.6.0")),
```
Comment (ShanMa1991) [inline Package.swift:28]: Should we change to the up to the minimum version?

**Review by ShanMa1991 — APPROVED** [review]

File: Package.swift (hunk 2 above)
Comment (1ec5) [reply Package.swift:28]: Yes, this should be a `from:` as well.

Comment (MaximAlien) [reply Package.swift:28]: Good catch. Will update.

File: Sources/MapboxCoreNavigation/NativeHandlersFactory.swift (hunk 1 above)
Comment (Udumft) [reply Sources/MapboxCoreNavigation/NativeHandlersFactory.swift:57]: It definitely used to. Debug messages were quite spammy, logging basically any NN method enter/leave. @mapbox/navnative did you update something here? v106 changelog does not mention logging changes.

Comment (mskurydin) [reply Sources/MapboxCoreNavigation/NativeHandlersFactory.swift:57]: These are the changes on the core-sdk side - the logging API in v22.0.0 is different.

Comment (Udumft) [reply Sources/MapboxCoreNavigation/NativeHandlersFactory.swift:57]: I see, but Maxim said that debug messages are still appearing even though log level is set higher. Do you see same behavior on your side? Should we ping core team?

**Pull request merged**